### PR TITLE
add ObsPyck recipe

### PR DESCRIPTION
**Pull request by megies**
This PR adds a recipe for the ObsPyck GUI application for seismogram analysis.

### CODE REVIEW

**Comment by megies** [comment]
CircleCI:
```
conda.resolve.NoPackagesFound: Packages/dependencies missing in current linux-64 channels: 
  - obspyck 0.2.0 py27_0 -> obspy 0.10.*
  - obspy 0.10.*
```

We have older packages of obspy at https://anaconda.org/obspy/obspy. I guess it's not possible to add another channel through the recipe, right?

**Comment by megies** [comment]
Not sure what the linter is complaining about, and I can't seem to trigger it again. Not sure what the problem with Appveyor is either, I'm skipping windows builds anyway.

From my point of view this is ready to go..

File: recipes/obspyck/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  # current codebase is still python 2
+  skip: True  # [not py27]
+  # never was tested/confirmed if it runs on win
+  skip: True  # [win]
```
Comment (megies) [inline recipes/obspyck/meta.yaml]: Ah, OK. Maybe the example recipe should get a comment on that..

**Comment by megies** [comment]
Seems everything is good to go now?

File: recipes/obspyck/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  # current codebase is still python 2
+  # never was tested/confirmed if it runs on win
```
Comment (ocefpaf) [inline recipes/obspyck/meta.yaml]: No need for these comments too IMO.

**Comment by ocefpaf** [comment]
Two minor comments that can be addressed in the feedstock if you agree with them.

Thanks @megies!

**Pull request merged**